src/components/search-bar.js

```diff
@@ -1,8 +1,27 @@
 import React, {Component} from 'react';
 
-export class SearchBar extends Component {
+export default class SearchBar extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {inputValue: ''};
+    }
+
+    searchForVideos() {
+        this.props.onSearch(this.state.inputValue);
+    }
 
     render() {
-        return <input onChange={event => console.log(event.target.value)}/>;
+        return (
+            <div className="search-bar row m-2">
```

Comment:
Maintain a separate file for HTML . What if this content keeps growing and growing ??